src/kili/services/label_import/__init__.py

```diff
@@ -63,3 +68,30 @@ def import_labels_from_files(  # pylint: disable=too-many-arguments
         model_name,
         is_prediction,
     )
+
+
+def import_labels_from_dict(kili, labels: List[Dict], label_type: LabelType):
+    """
+    Imports labels from a list of dictionaries
+    """
+    LabelsValidator(labels=labels)
+    labels_data = [
+        {
+            "jsonResponse": dumps(label.get("json_response")),
+            "assetID": label.get("asset_id"),
+            "secondsToLabel": label.get("seconds_to_label"),
+            "modelName": label.get("model_name"),
+            "authorID": label.get("author_id"),
+        }
+        for label in labels
+    ]
+    batch_generator = pagination.batch_iterator_builder(labels_data)
+    result = []
+    for batch_labels in batch_generator:
+        variables = {
+            "data": {"labelType": label_type, "labelsData": batch_labels},
+            "where": {"idIn": [label["assetID"] for label in batch_labels]},
+        }
+        batch_result = kili.auth.client.execute(GQL_APPEND_MANY_LABELS, variables)
+        result.extend(format_result("data", batch_result, Label))
```

Comment:
`List[Label]` ? not sure

Reply:
no, the array feature is handled in the format_result already is exploited in the format_result